recipe_scrapers/southernliving.py

```diff
@@ -71,26 +71,24 @@ def image(self):
         return lnk  # if image else None
 
     def ingredients(self):
-        ingredientsOuter = self.soup.find("div", {"class": "ingredients"})
+        ingredientsOuter = self.soup.find("ul", {"class": "ingredients-section"})
         ingGroup = []
         lines = ingredientsOuter.findAll("li")
         for li in lines:
-            ingGroup.append(li.get_text())
+            ingGroup.append(normalize_string(li.label.get_text()))
+
+        for i in ingGroup:
+            print(">>>", i)
```

Comment:
Is this a debugging `print()`?

Reply:
@bfcarpio fixed. Whoops, removed debug code. I also refactored the list comprehension